src/components/form/Method.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styles from '../../styles/method.css';
+
+export default class Method extends Component {
```

Comment:
each radio button should not be its own component. Rather, "method" should be stored in state and changing the radio button selection updates state